### DISCUSSION TITLE
increase limitations on User override

### DISCUSSION THREAD

**Discussion by BrandMChaos** [proginosko/LeechBlockNG#450] [Ideas]
- Add a maximum 'override activation' threshold to allow for flexibility when the user REALLY needs it
- Option to only override a block for X minutes as defined by the user

**Comment by proginosko** [comment]
If I understand these both correctly, they're already available features:
- See "Allow no more than __ override(s) every __" under General options.
- Leave "Allow blocking to be suspended for __ minute(s)" option blank to specify override duration at time of activation.
